docs/environment-variables.md

```diff
@@ -0,0 +1,59 @@
+# Environment variables
+
+In development, environment variables are resolved from the .env file
+In production you are expected to set environment variables during deployment (e. g. using Vault)
+
+## List of environment variables
+
+Legend:
+
+- M - mandatory
+- O - optional (default value)
+
+### app
+
+- (M) `APP_ENV` - environment, in which app is running. Supported values: 'production' | 'development' | 'staging'
```

Comment:
🟢 Just a nit, but I would prefer having no legend, no mandatory marker and just a prefix `(OPTIONAL)` for optional envs. Currently M and O take the same space, are mixed together (not grouped) and are a bit hard to spot from a quick glance.